lib/util.js

```diff
@@ -509,6 +509,10 @@ function formatValue(ctx, value, recurseTimes) {
                    'byteOffset',
                    'buffer');
     }
+  } else if (binding.isSharedArrayBuffer(value)) {
+    braces = ['{', '}'];
+    keys.unshift('byteLength');
+    visibleKeys.byteLength = true;
```

Comment:
Just a suggestion but I think you can join this with the `isArrayBuffer` case using `||`.